sources/parsing/parsing.h

```diff
@@ -0,0 +1,34 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   parsing.h                                          :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/11 16:40:11 by taretiuk          #+#    #+#             */
+/*   Updated: 2024/10/16 18:51:51 by taretiuk         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef PARSING_H
+# define PARSING_H
+# include "../minishell.h"
+
+# include <stddef.h>
+# include <stdlib.h>
+
+typedef struct s_delim
```

Comment:
These types should be defined in libft/strings.h, not here

Reply:
ok, but why?